packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx

```diff
@@ -282,7 +282,9 @@ export const CommentsItem = ({ treeOptions, comment, nestingLevel=1, isChild, co
           </div>
         )}
 
-        {showPostTitle && !isChild && hasPostField(comment) && comment.post && <Link className={classes.postTitle} to={commentGetPageUrlFromIds({postId: comment.postId, commentId: comment._id, postSlug: ""})}>{comment.post.title}</Link>}
+        {showPostTitle && !isChild && hasPostField(comment) && comment.post && <LWTooltip tooltip={false} title={<PostsPreviewTooltipSingle postId={comment.postId}/>}>
+            <Link className={classes.postTitle} to={commentGetPageUrlFromIds({postId: comment.postId, commentId: comment._id, postSlug: ""})}>{comment.post.title}</Link>
+          </LWTooltip>}
```

Comment:
Note that this is incorrectly indented; a close tag should always be at the same indent level as the line that contains the corresponding open tag, even if there's a bunch of random stuff before the open tag.